examples/01-getting-started.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<p>What I'm about to ask might be obvious to someone with experience, but is there anything to say about your choices for the embedding dimension or the values for the MLP blocks that is related to the characteristics of the dataset?</p><p> </p><p>Again, could be obvious to someone more experienced, but is there anything to say about the appropriateness of <code>BinaryClassifcationTask</code> to the dataset?</p><p> </p><p>You <em>could</em> add a one-liner after the code block to provide a teaser about <code>Tags.TARGET</code> and hint about what it does and how it helps speed the modelling effort.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I think we provide more details in the next notebooks about these functionalities.</p>